cypress/e2e/gui/homepage_gui_tests.cy.js

```diff
@@ -0,0 +1,37 @@
+
+describe('EngageSphere', () => {
+
+    // Constants used across multiple tests
+    const today = new Date(2024, 6, 9) // July 9, 2024
+    const formattedDate = today.toDateString()
+
+    beforeEach(() => {
+        cy.visit('/')
+        cy.clock(today.getTime())
+    })
+
+    it('Homepage renders the header with an h1 and theme toggle', () => {
```

Comment:
What about this?

```suggestion
    it('renders the header with an h1 and theme toggle', () => {
```

This is a single-page application. So, there's only a page, no other.